overview/templates/overview/candidate_detail.html

```diff
@@ -336,6 +336,92 @@ <h2 id="questionnaires">Questionnaires</h2>
     </div>
 </div>
 
+{% if user.is_staff %}
+<div class="row">
+    <div class="col-md-6">
+        <h2 id="">Quotes</h2>
+            {% for quote in candidate.quote_set.all %}
+                {% if quote.display %}
+                    <blockquote> 
+                        <p>{{ quote.quote|markdown }}</p>
+                            <footer>
+                                {% if quote.by %} {{ quote.by }} {% else %} {{ candidate.fullname }}{% endif %}{% if quote.cite %}, <cite>{{ quote.cite }}</cite>{% endif %}
+                            </footer>
+                        </blockquote>
+                {% endif %}
+            {% endfor %}
+    </div>
+    <div class="col-md-6">
+        <h2 id="">Press</h2>
+        {% if candidate.pressarticlecandidate_set.all %}
+            <ul>
+            {% for article in candidate.pressarticlecandidate_set.all %}
```

Comment:
I would probably make the PressArticle have a manager and do something like

```
class PressArticleManager(models.Manager):
    def public_own(self):
         assert isinstance(self.instance, Candidate)
         return super().get_queryset().filter(public=True, candidate_is_the_author=True):

    def public_not_own(self):
         assert isinstance(self.instance, Candidate)
         return super().get_queryset().filter(public=True, candidate_is_the_author=True):
```

but if we throw all of this in front of a cache, it doesn't matter

Reply:
this is neat but I'll wait on it for now.